tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1388,6 +1399,46 @@ static __always_inline int del_retval(u32 event_id)
     return 0;
 }
 
+static __always_inline int save_sockfd(u32 sockfd)
+{
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+    u32 tgid;
+    tgid = (u32)((pid_tgid & 0xFFFFFFFF00000000LL) >> 32);
```

Comment:
```suggestion
    u32 tgid = pid_tgid >> 32;
```

Reply:
we can assign directly to u32 to get the pid.

"By directly setting this to a u32, we discard the upper 32 bits."
(from here: https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md)